scripts/store.js

```diff
@@ -23,16 +24,35 @@ export class Store extends EventEmitter {
   load() {
     return this.collection.list()
       .then(res => {
-        this.state.items = res.data;
+        this.state.projects[this.project].items = res.data;
         this.emit("change", this.state);
       })
       .catch(this.onError.bind(this));
   }
 
+  setAuth() {
+    var auth = localStorage.getItem("Authorization");
+    this.collection.api.optionHeaders.Authorization = 'Basic ' + auth;
+  }
+
+  setProject(project) {
```

Comment:
I'm not sure there is a lot of value in having the app know the fact that there are multiple projects (appart from the fact that it would be easy to switch from one to one another). But IMO this is a different approach that what's proposed here.